GoedeDoelenHelpen/ClientApp/package.json

```diff
@@ -30,9 +30,11 @@
     "@angular/platform-server": "6.0.0",
     "@angular/router": "6.0.0",
     "@nguniversal/module-map-ngfactory-loader": "^5.0.0-beta.5",
+    "@swimlane/ngx-charts": "^8.0.0",
```

Comment:
@myself will update documentation why we have chosen specific libraries and solution

Reply:
Very good!